dev/build_release.py

```diff
@@ -266,35 +266,56 @@ def publish_to_bintray(self, source, package, version, path, debian_tags=''):
         try:
             result = urllib2.urlopen(put_request, data)
         except HTTPError as put_error:
-            if put_error.code != 400:
+            if put_error.code == 409:
```

Comment:
Looks like this is actually the suggested workaround: https://github.com/pledbrook/lazybones/issues/111#issuecomment-46432234